tests/functions/dates.test.ts

```diff
@@ -1012,3 +1012,27 @@ describe("Number of digits in year/month/days in format", () => {
     expect(parseDateTime("1-3-2002", locale)).toMatchObject({ format: "m-d-yyyy" });
   });
 });
+
+describe("Datetime", () => {
+  test("isoWeekNumber", () => {
```

Comment:
I used https://www.epochconverter.com/weeks/2024 to ensure that this test is correct